pom.xml

```diff
@@ -410,6 +410,92 @@
         </skipTestResourceEnforcement> <!-- Test resource profile must be enabled unless using -DskipTests etc -->
     </properties>
 
+    <dependencyManagement>
+        <dependencies>
+            <dependency>
+                <groupId>io.netty</groupId>
+                <artifactId>netty-all</artifactId>
+                <version>${netty.version}</version>
+            </dependency>
+            <dependency>
+                <groupId>io.netty</groupId>
```

Comment:
You can get rid of this version of netty. It will just clash.

Reply:
@ShamsUlAzeem  try to focus on the UI module as much as possible. That's where the major clash we need to fix is.